### PR TITLE
[Term Entry] setMinimalDaysInFirstWeek()

### PR DESCRIPTION
**Pull request by roberanegussie**
### Description

I have added a new term entry for the calendar class by creating a new file setMinimalDaysInFirstWeek used to set the minimal required days in first week of the year. This is my first time contributing to this repo so took time to make sure I did it well. 
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/setMinimalDaysInFirstWeek/setMinimalDaysInFirstWeek.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```pseudo
+calendar.setMinimalDaysInFirstWeek(int day)
+```
+
+This method takes only one argument, `int day`, which is an integer constant representing the number of days which will be set as the minimal days required in the first week of the year.
+
+## Example
+
+In the following example, `.setMinimalDaysInFirstWeek()` is used to set the minimal days in required in first week of the year. This example will set the minimal days required in the first week of the year to be 2 days.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/setMinimalDaysInFirstWeek/setMinimalDaysInFirstWeek.md:27]:
```suggestion
In the following example, `.setMinimalDaysInFirstWeek()` is used to set the minimal days required in the first week of the year. 
```

File: content/java/concepts/calendar/terms/setMinimalDaysInFirstWeek/setMinimalDaysInFirstWeek.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      // print the minimak number of days required in the first week
+      int min = cal.getMinimalDaysInFirstWeek();
+      System.out.println("Minimal Days in Week: " + min);
+    }
+    
+}
+```
+
+The output of the above code will be set the minimal days required in first week of the year as 2 days:
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/setMinimalDaysInFirstWeek/setMinimalDaysInFirstWeek.md:50]:
```suggestion
The output of the above code will be:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@roberanegussie I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by roberanegussie** [comment]
@caupolicandiaz  I have updated the PR based on the required suggestions please review.

**Review by caupolicandiaz — APPROVED** [review]
@roberanegussie Looks good, this is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@roberanegussie Looks good.

**Pull request merged**